src/client/styles/base.scss

```diff
@@ -3,6 +3,11 @@ body, html, #app, * {
 	padding: 0;
 	box-sizing: border-box;
 }
+#app {
+	height: 100vh;
+	position: relative;
+	overflow: hidden;
```

Comment:
A reason for hiding the app overflow? An image maybe?

Reply:
You're right, Now this hidden property is in LoginTemplate